docs/emitters/typespec-autorest/reference/emitter.md

```diff
@@ -119,3 +119,9 @@ This extension is meant for debugging and should not be depended on.
 **Type:** `boolean`
 
 Create read-only property schema for lro status
+
+### `suppress-lro-options`
```

Comment:
```suggestion
### `omit-lro-options`
```
would that name be better and more inline with some other options?

Reply:
I am cool with that